worldwind/src/commonMain/kotlin/earth/worldwind/render/program/TriangleShaderProgram.kt

```diff
@@ -9,31 +9,46 @@ import earth.worldwind.util.kgl.KglUniformLocation
 open class TriangleShaderProgram : AbstractShaderProgram() {
     override var programSources = arrayOf(
         """
+            #version 300 es
```

Comment:
Should we add ES 3.0 declaration in Android manifest also? Or we already have it there?

Reply:
we will need to rewrite all other shader, probably should be done in separate pr if ever